### DISCUSSION TITLE
Возможность есть некоторые ингридиенты

### DISCUSSION THREAD

**Discussion by VasilievNik** [octothorp-team/public-dobrograd#702] [Предложения]
Я человек кулинарии, что в жизни, что в Доброграде. И, как по мне, невозможность съесть некоторую пищу, помеченную как "Это ингредиент - его нельзя съесть, но можно приготовить", слегка уменьшает иммерсивность РП процесса. 
Пусть сырая еда(ингредиенты) и давала бы ничтожно-малую прибавку к уровню голода, но это это значительно могло бы поднять реализм и, как уже было сказано, иммерсивность, дав возможность появления множеству интересных ситуаций!

**Comment by LENFYS** [comment]
За
